packages/core/src/new-api/types/deployment-loader.ts

```diff
@@ -0,0 +1,19 @@
+import { Artifact, BuildInfo } from "./artifact";
+import { Journal } from "./journal";
+
+/**
+ * Read and write to the deployment storage.
+ *
+ * @beta
+ */
+export interface DeploymentLoader {
+  journal: Journal;
+  initialize(): Promise<void>;
+  loadArtifact(storedArtifactPath: string): Promise<Artifact>;
```

Comment:
I don't understand which path this receives, the internal to the deployment or the one that the Resolver uses?

Reply:
It uses whatever path is returned from `deploymentLoader.storeArtifact`.